hail/python/hail/methods/impex.py

```diff
@@ -1643,6 +1645,11 @@ def import_matrix_table(paths,
     force_bgz : :obj:`bool`
         If ``True``, load **.gz** files as blocked gzip files, assuming
         that they were actually compressed using the BGZ codec.
+    sep : :obj:`str`
+        This parameter is a deprecated name for `delimiter`, please use that
+        instead.
+    delimiter : :obj:`str`
+        A single character string which separates values in the file.
 
     Returns
     -------
```

Comment:
I was expecting to see something like this:

```
assert one of sep and delimiter is defined, but not both
use value from whichever is defined
```

Reply:
Wow good point, I must've been falling asleep when I wrote this. Fixed.

Reply:
Do we need tests for this?